### ISSUE TITLE
“Dual collapse” annotation treats coordinate-dependent diagnostics as an identified causal mechanism

### ISSUE THREAD

**Issue by microprediction** [microprediction/latentreasoning#100]
## Reviewed source

Default-branch commit: [`88d1db6b1c014f641f67ec4b84ec1f51e8f0c43b`](https://github.com/microprediction/latentreasoning/commit/88d1db6b1c014f641f67ec4b84ec1f51e8f0c43b)

Affected text:

- [`training.html` lines 116–125](https://github.com/microprediction/latentreasoning/blob/88d1db6b1c014f641f67ec4b84ec1f51e8f0c43b/training.html#L116-L125) says Chen et al. “measure [what outcome supervision loses] directly,” that the gradient norm “decays along the chain until positions two through six are effectively inactive,” and that the states “drift out of the region the token embeddings occupy.”
- [`bibliography.html` lines 1189–1192](https://github.com/microprediction/latentreasoning/blob/88d1db6b1c014f641f67ec4b84ec1f51e8f0c43b/bibliography.html#L1189-L1192) says the paper “identifies a dual collapse, gradient attenuation together with representational drift, as why outcome-only supervision of latent chains is weak.”

Primary source: Chen et al., [*What Makes Effective Supervision in Latent Chain-of-Thought: An Information-Theoretic Analysis*, arXiv:2606.20075v1](https://arxiv.org/html/2606.20075), especially [§3](https://arxiv.org/html/2606.20075#S3), [§4.1](https://arxiv.org/html/2606.20075#S4.SS1), and [Appendices A and C](https://arxiv.org/html/2606.20075#A1).

I searched open and closed issues for the identifier, title, “dual collapse,” gradient attenuation, manifold drift, and outcome supervision; I found no issue covering this paper.

## The accuracy results are real; the causal diagnosis is not identified

The reported endpoint numbers are accurately transcribed. In the main GPT-2/GSM8K-Aug experiment, outcome-only latent variants peak at 9.8/18.3%, progressive step supervision at 31.2%, and generative reconstruction at 39.1% versus 43.1% explicit CoT.

But the site turns the paper’s proposed mechanism into a direct measurement of what outcome supervision generally loses. The paper does not causally isolate either half of the “dual collapse”:

1. **Small position-wise gradients do not show that positions are inactive or that credit decayed along a causal path.** The paper measures

```
G(t) = ‖∂L/∂L_t‖₂
```

and observes a large norm at the first latent position and smaller norms at positions 2–6. It does not ablate those positions, patch them, or measure their mediated effect on the answer.

Gradient magnitude is also coordinate-dependent. For any nonzero scale `a_t`, define

```
L̃_t = a_t L_t
F̃(L̃_t) = F(L̃_t / a_t)
```

The composed model computes exactly the same function, while

```
‖∂L/∂L̃_t‖₂ = |a_t|⁻¹ ‖∂L/∂L_t‖₂.
```

Choosing large `a_t` for positions 2–6 makes their gradients arbitrarily small without changing their computation or causal importance. A raw cross-position norm profile can be a useful optimization diagnostic, but it is not a representation-invariant inactivity test.

2. **PCA distance from token embeddings does not establish semantic or representational loss.** The “manifold drift” evidence is a visualization of latent hidden states across epochs relative to explicit-CoT embeddings. An invertible change of basis `Q`, with the next consumer changed from `F(L)` to `F(Q⁻¹L̃)`, preserves the model function and all mutual information while moving `L̃ = QL` arbitrarily in the plotted coordinate system. Continuous latent reasoning need not remain geometrically close to token embeddings to be meaningful. The paper runs no causal intervention showing that the observed distance causes the accuracy gap.

3. **The supervision comparison changes several things at once.** The process-supervised condition supplies explicit suffix targets, uses a progressive curriculum, changes the number of latent tokens by stage, and resets AdamW at every stage. The paper’s own Table 3 reports that omitting the reset lowers PS-Latent from 31.2% to 24.7%, and §4.1 attributes part of the gain to an “optimization shock.” Therefore the comparison does not isolate step-level supervision, let alone prove that it works specifically by repairing the plotted gradient/PCA phenomena.

4. **The mechanistic scope is narrower than the endpoint replication.** The main diagnostic experiment uses one GPT-2 124M setup, six latent positions, GSM8K-Aug, and a checkpoint first pretrained on full CoTs for 11 epochs. Appendix C reproduces the *accuracy pattern* on Llama-3.2-1B and ProntoQA, but does not reproduce the gradient-norm and PCA diagnostics there. The paper itself lists model scale, task specificity, and probe capacity as limitations.

The defensible inference is that, in these experiments, outcome-only latent tokens did not improve over answer-only training and correlated with a skewed gradient profile and geometric separation from an explicit-CoT reference. That is valuable evidence. It does not show that the latter two observations are the causes, that positions 2–6 are inactive, or that token-embedding proximity is required for latent reasoning.

## Suggested correction

For example:

> In Chen et al.’s GPT-2/GSM8K-Aug setup, outcome-only latent training reaches 9.8–18.3%, versus 31.2% with a progressive explicit-suffix curriculum and 39.1% when that curriculum is combined with step reconstruction (43.1% explicit CoT). The outcome-only run also shows a strongly skewed position-wise gradient profile and PCA separation from explicit-CoT embeddings. These are correlational, coordinate-dependent diagnostics rather than causal tests of latent-position inactivity or semantic drift; the supervised comparison also changes curriculum and optimizer resets. Appendix experiments reproduce the endpoint gap on Llama-3.2-1B and ProntoQA, not the proposed mechanism.

The bibliography annotation should likewise call “dual collapse” the paper’s diagnosis in its tested setup, not an identified general explanation.

## Bibliographic metadata

The same entry also has incorrect initials. The paper’s authors are Xinghao Chen, Chak Tou Leong, Wenjin Guo, Jian Wang, Wenjie Li, and Xiaoyu Shen, so the entry should read **Chen, X.; Leong, C. T.; Guo, W.; Wang, J.; Li, W.; Shen, X.**, not Chen, S.; Guo, Y.; Li, X.; Shen, Z.

**Comment by microprediction** [comment]
Confirmed and applied in 0e66055. The accuracy figures were right, but §3 supports the gradient attenuation and manifold drift claims only with Fig. 2(b) and a PCA plot, with no ablation or patching. Table 3 shows PS-Latent falls from 31.2% to 24.7% without optimizer resets, so that comparison is not step supervision alone. Appendix C repeats the accuracy tables on Llama-3.2-1B and ProntoQA but not the diagnostics. The training page and the bibliography now present dual collapse as the authors' diagnosis, and the author initials match the arXiv listing.
